src/main/java/fi/otavanopisto/kuntaapi/server/controllers/IdentifierController.java

```diff
@@ -65,6 +78,106 @@ public Identifier findIdentifierByTypeSourceAndKuntaApiId(String type, String so
+    List<OrganizationId> result = new ArrayList<>(organizationIds.size());
+    
+    for (String organizationId : organizationIds) {
+      result.add(new OrganizationId(source, organizationId));
+    }
+    
+    return result;
+  }
+  
+  public List<PageId> listOrganizationPageIdsBySource(OrganizationId organizationId, String source) {
+    List<String> pageIds = listSourceIdsByOrganizationIdAndSourceAndType(organizationId, source, IdType.PAGE.toString());
+    List<PageId> result = new ArrayList<>(pageIds.size());
+    
+    for (String pageId : pageIds) {
+      result.add(new PageId(organizationId, source, pageId));
+    }
+    
+    return result;
+  }
+  
+  public List<BannerId> listOrganizationBannerIdsBySource(OrganizationId organizationId, String source) {
+    List<String> bannerIds = listSourceIdsByOrganizationIdAndSourceAndType(organizationId, source, IdType.BANNER.toString());
+    List<BannerId> result = new ArrayList<>(bannerIds.size());
+    
+    for (String bannerId : bannerIds) {
+      result.add(new BannerId(organizationId, source, bannerId));
+    }
+    
+    return result;
+  }
+
+  public List<NewsArticleId> listOrganizationNewsArticleIdsBySource(OrganizationId organizationId, String source) {
+    List<String> newsArticleIds = listSourceIdsByOrganizationIdAndSourceAndType(organizationId, source, IdType.NEWS_ARTICLE.toString());
+    List<NewsArticleId> result = new ArrayList<>(newsArticleIds.size());
+    
+    for (String newsArticleId : newsArticleIds) {
+      result.add(new NewsArticleId(organizationId, source, newsArticleId));
+    }
+    
+    return result;
+  }
+
+  public List<MenuId> listOrganizationMenuIdsBySource(OrganizationId organizationId, String source) {
+    List<String> menuIds = listSourceIdsByOrganizationIdAndSourceAndType(organizationId, source, IdType.MENU.toString());
+    List<MenuId> result = new ArrayList<>(menuIds.size());
+    
+    for (String menuId : menuIds) {
+      result.add(new MenuId(organizationId, source, menuId));
+    }
+    
+    return result;
+  }
+
+  public List<TileId> listOrganizationTileIdsBySource(OrganizationId organizationId, String source) {
+    List<String> tileIds = listSourceIdsByOrganizationIdAndSourceAndType(organizationId, source, IdType.TILE.toString());
+    List<TileId> result = new ArrayList<>(tileIds.size());
+    
+    for (String tileId : tileIds) {
+      result.add(new TileId(organizationId, source, tileId));
+    }
+    
+    return result;
+  }
+  
+  private List<String> listSourceIdsBySource(String source, String type) {
+    List<Identifier> identifiers = identifierDAO.listBySourceAndType(source, type);
+    List<String> result = new ArrayList<>(identifiers.size());
+    
+    for (Identifier identifier : identifiers) {
+      result.add(identifier.getSourceId());
+    }
+    
+    return result;
+  }
+
+  private List<String> listSourceIdsByOrganizationIdAndSourceAndType(OrganizationId organizationId, String source, String type) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "listSourceIdsByOrganizationIdAndSourceAndType" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)